### ISSUE TITLE
[Bug]: Heartstone crashes while opening

### ISSUE THREAD

**Issue by whitenight48** [PlayCover/PlayCover#1693]
### Describe the bug

As the title shows.

### Steps to reproduce

MacBook Air with M2 chip, system sequoia version 15.0.

Crashes while opening. 

### Expected behaviour

functioning well?

### What version of PlayCover are you using?

3.1.0

### What version of macOS are you using?

Other (please specify)

### Issue Language

- [X] Yes my issue is written in English

**Comment by whitenight48** [comment]
EMMM by the way heartstone for CN version

**Comment by xiaoheiCat** [comment]
You can try to remove the PlayTools from the game. The game will be launched, but then it cannot download resources. Tried on Macbook Air, heartstone of CN version.

**Comment by whitenight48** [comment]
> You can try to remove the PlayTools from the game. The game will be launched, but then it cannot download resources. Tried on Macbook Air, heartstone of CN version.

Yeah i tried it just exactly what you said, and the result expertly turns out to be the same (bruh)

**Comment by PandaPYHNN** [comment]
Im using the Mac mini M4 2024, I also met the crash issue for opening the CN version of Hearthstone at first.
After checked this issue and see the xiaoheiCat's suggestion, I remove the PlayTools. 
Then it can successfully launch.
Now playing @144 hz.

PlayCover: Version 3.1.0 (856)
Hearthstone: 32.0.217964
